src/Extension/FluentExtension.php

```diff
@@ -511,4 +514,63 @@ protected function getManipulationRecordID($updates)
         }
         return null;
     }
+
+    /**
+     * Add the current locale's URL segment to the start of the URL
+     *
+     * @param  string &$base
+     * @param  string &$action
+     * @return null
+     */
+    public function updateRelativeLink(&$base, &$action)
```

Comment:
We should move this to FluentSiteTreeExtension, since it's sitetree specific.

Reply:
Also, we can use `updateLink` now, if you want to merge the domain with relativelink there. :)